modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -166,6 +166,13 @@ protected void configure() {
             }
         }
 
+        // hack for new selenium-chrome image that contains Chrome 92
+        // If not disabled, container startup will fail in most cases and consume excessive amounts of CPU
+        if (seleniumVersion.equals("3.141.59")  && capabilities instanceof ChromeOptions) {
```

Comment:
since `--disable-gpu` is available in Chrome since 2014 or so, perhaps we should do it unconditionally?
I do not think there are many users out there who even have their GPUs proxied to Docker

Reply:
I would effectively be fine with it. The need for this flag was seemingly deprecated in 2016 or so, but I don't think having it in there unconditionally (for `ChromeOptions`) will hurt us in any way.